modules/apps/frontend-theme-locos/locos-theme/src/css/_banner.scss

```diff
@@ -0,0 +1,93 @@
+%astyle {
+  color: #999990;
+  font-family: 'Poppins', sans-serif;
```

Comment:
Do we need additional fall back fonts in case ```Poppins``` does not work or will any generic ```sans-serif``` font be ok?

Reply:
I suppose anything would be fine.  I shall remove that!